### ISSUE TITLE
Have more convenient way to include debian packages which are not in GL repos

### ISSUE THREAD

**Issue by fwilhe** [gardenlinux/builder#31]
**What would you like to be added**:

I'd like to have a better way to add packages which are not included in the gardenlinux debian repos. I do understand that there is a reason why the gl repos have very few packages and we don't need to pollute the repos with packages that are only sometimes needed for debugging, but having a simple way to add packages from the upstream debian testing repos would be nice.

My current workaround is described here https://github.com/gardenlinux/gardenlinux/issues/1708#issuecomment-1698707150

What I'm thinking of would be something like this: The builder could recognize if a feature has a `pkg.upstream.include` file which has the same format as `pkg.include` but install the given packages from the debian testing repo automatically.

**Why is this needed**:

Not sure if it is needed and it surely has downsides too (people might abuse this to install non-gl maintained packages in 'prod' images), but for my development work not being limited by gl's debian repos would be helpful.

**Comment by Vincinator** [comment]
I agree, we should think about including a convenient way to include packages from other apt repositories than repo.gardenlinux.io
 
There were requests in the past, and I am sure there will be more requests in the future. 
Especially now, that we offer the builder and switched to the design of "bring your own features".  

For now, I will document pretty much your workaround, this is the current intended way. 
We are waiting that the gardenlinux-repo-manager is finished. Once it is done, we will implement a new convenient way that is based on that gardenlinux-repo-manager. The idea is, that the user of the builder only needs to specify a list of packages that should be installed from other sources.

**Comment by sramapu** [comment]
closing due to no activity for more than 2y
